ansible_collections/arista/avd/examples/l2ls-fabric/documentation/fabric/DC1-documentation.md

```diff
@@ -22,12 +22,12 @@
 
 | POD | Type | Node | Management IP | Platform | Provisioned in CloudVision | Serial Number |
 | --- | ---- | ---- | ------------- | -------- | -------------------------- | ------------- |
-| DC1_FABRIC | l2leaf | LEAF1 | 172.16.100.105/24 | cEOS-LAB | Provisioned | - |
-| DC1_FABRIC | l2leaf | LEAF2 | 172.16.100.106/24 | cEOS-LAB | Provisioned | - |
-| DC1_FABRIC | l2leaf | LEAF3 | 172.16.100.107/24 | cEOS-LAB | Provisioned | - |
-| DC1_FABRIC | l2leaf | LEAF4 | 172.16.100.108/24 | cEOS-LAB | Provisioned | - |
-| DC1_FABRIC | l2spine | SPINE1 | 172.16.100.101/24 | cEOS-LAB | Provisioned | - |
-| DC1_FABRIC | l2spine | SPINE2 | 172.16.100.102/24 | cEOS-LAB | Provisioned | - |
+| DC1 | l2leaf | LEAF1 | 172.16.100.105/24 | vEOS-lab | Provisioned | - |
```

Comment:
again, I don't like the change from cEOS-lab to vEOS-lab. Please keep cEOS!

Reply:
As discussed internally cEOS is the way to go as the default choice for everything.